pkg/asset/machines/aws/machinesets.go

```diff
@@ -67,13 +66,14 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 				Template: machineapi.MachineTemplateSpec{
 					ObjectMeta: metav1.ObjectMeta{
 						Labels: map[string]string{
-							"machine.openshift.io/cluster-api-machineset":   name,
-							"machine.openshift.io/cluster-api-cluster":      clusterID,
-							"machine.openshift.io/cluster-api-machine-role": role,
-							"machine.openshift.io/cluster-api-machine-type": role,
+							"machine.openshift.io/cluster-api-machineset": name,
```

Comment:
this will  never pass the tests https://github.com/openshift/origin/blob/master/test/extended/machines/workers.go#L23
we'd need to update the test to fetch from the node